src/main/java/frc/team2412/robot/subsystems/IntakeSubsystem.java

```diff
@@ -77,19 +85,28 @@ public void intakeStop() {
 	}
 
 	public GamePieceType detectType() {
-		// if () {
-		//     return GamePieceType.CUBE;
-		// }
-		// else if () {
-		//     return GamePieceType.CONE;
-		// }
-
+		if (colorSensor.getColor().equals(INTAKE_CUBE_COLOR)) {
```

Comment:
Do we know if the color returned from the sensor will always be exactly equal to `INTAKE_CUBE_COLOR` or `INTAKE_CONE_COLOR`? I'm thinking about what would happen if the returned color changes due to lighting conditions, proximity, etc.